raiden-dapp/src/components/AppHeader.vue

```diff
@@ -52,16 +52,22 @@
             </template>
             <span>{{ defaultAccount }}</span>
           </v-tooltip>
-          <v-tooltip v-model="copied" bottom dark activator="#copyBtn">
-            <span>{{ $t('app-header.copy-success') }}</span>
+          <v-tooltip bottom dark close-delay="1500">
+            <template v-slot:activator="{ on }">
```

Comment:
Can you update to [shorthand](https://vuejs.org/v2/guide/components-slots.html#Named-Slots-Shorthand) form here? (eslint rule is not yet released, I will included it as soon as the version that includes it is released)